lib/guardian/plug.ex

```diff
@@ -212,9 +212,9 @@ if Code.ensure_loaded?(Plug) do
     def remember_me(conn, mod, resource, claims, opts) do
       with type <- Keyword.get(opts, :token_type, "refresh"),
            opts <- Keyword.put(opts, :token_type, type),
-           key <- Pipeline.fetch_key(conn, opts),
+           key <- Pipeline.fetch_key(conn, opts) |> token_key() |> Atom.to_string(),
            {:ok, token, _claims} <- Guardian.encode_and_sign(mod, resource, claims, opts) do
-        put_resp_cookie(conn, key, token)
+        put_resp_cookie(conn, key, token, [secure: true, max_age: 1000])
```

Comment:
To get the max age here you can extract the expiry from the claims (allow for expiry not being there). It might look something like:

```elixir
@default_cookie_options [secure: true, max_age: 60 * 60 * 24 * 7 * 4]

def remember_me(conn, mod, resource, claims, opts) do
  # snip
  {:ok, token, full_claims} <- Guardian.encode_and_sign(mod, resource, claims, opts)
   
  cookie_opts = cookie_options(mod, full_claims)
  put_resp_cookie(conn, key, token, cookie_opts)
  # snip
end

defp cookie_options(mod, %{"exp" => timestamp}) do
  max_age = timestamp - Guardian.timestamp()
  [max_age: max_age] ++ cookie_options(mod, %{})
end

defp cookie_options(mod, claims) do
  mod.config(:cookie_options, []) ++ @default_cookie_options
end
```

This would give an opportunity to set cookie options in config.